src/Tokenizer.js

```diff
@@ -164,3 +171,38 @@ export function identifier (index) {
 
 	return slice(index, caret())
 }
+
+/**
+ * @return {number}
+ */
+function _variable() {
```

Comment:
those names are ofc not perfect, i couldnt really figure out how to name them based on the naming scheme in stylis, any suggestions?